doc/en/announce/release-7.0.0.rst

```diff
@@ -0,0 +1,67 @@
+pytest-7.0.0
+=======================================
+
+pytest 7.0.0 has just been released to PyPI.
+
+This is a bug-fix release, being a drop-in replacement. To upgrade::
```

Comment:
i think the wrong template was picked as there is a rc

Reply:
Ouch, good catch... Will take a look ASAP.